### ISSUE TITLE
Crashes Despite Poetry Being In Use

### ISSUE THREAD

**Issue by Kurt-von-Laven** [Lucas-C/pre-commit-hooks-safety#14]
I have run `poetry install`, but still see this error message.

```
- hook id: python-safety-dependencies-check
- exit code: 2

usage: safety_check
       [-h]
       [--full-report]
       [--short-report]
       [--ignore IGNORE]
       files
       [files ...]
safety_check: error: Unsupported build tool: this pre-commit hook currently only handles pyproject.toml with Poetry
```

**Comment by Lucas-C** [comment]
Is your repository public? If so, could share it here in order for me to help you?

Currently this hook check for files with the following pattern:
https://github.com/Lucas-C/pre-commit-hooks-safety/blob/master/.pre-commit-hooks.yaml#L6

What files do you have in your repo that matches `pyproject\.toml|requirements`?

**Comment by Kurt-von-Laven** [comment]
Unfortunately no, but I appreciate the offer and am happy to provide more details. We have a pyproject.toml and no requirements file.

**Comment by Kurt-von-Laven** [comment]
This isn't our issue, but one issue I see with the logic is that there could be an inline comment after `[tool.poetry]`, which would be best addressed by using a spec-compliant toml parser.

**Comment by Lucas-C** [comment]
Could you share your `pyproject.toml` file?

**Comment by Kurt-von-Laven** [comment]
Oh, I see now that the `trim()` call was [not yet added in v1.2.0](https://github.com/Lucas-C/pre-commit-hooks-safety/blob/v1.2.0/pre_commit_hooks/safety_check.py#L43-L44). Upgrading to the unreleased tag v1.2.1 addresses the issue assuming no inline comments. Thank you for the help!

**Comment by Lucas-C** [comment]
Glad that simply upgrading solved your issue.

I released a new minor version handle the case of a comment on the `[tool.poetry]` line: https://github.com/Lucas-C/pre-commit-hooks-safety/commit/c20e8a8125a5a51997164feb846a7171b50b5fe6

**Comment by Kurt-von-Laven** [comment]
@Lucas-C, it's worth noting that you haven't actually released v1.2.1 or v1.2.2, only tagged them, which typically signals that the maintainer doesn't yet recommend upgrading on account of known bugs, insufficient testing, etc.
